### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by BALAJI141031**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -16,9 +20,14 @@
   --hover-color: rgb(212, 250, 250);
   --anchor-el-color: #000000;
   --footer-color: #fff;
+  --warn-color:rgb(255, 150, 14);
+  --danger-color:red;
+  --success-color:palegreen;
```
Comment (mayuresh-tech) [inline css/main.css:25]: You can use variables

File: css/main.css [hunk 2]
```diff
@@ -16,9 +20,14 @@
...
+
   --pale-white:#fff;
   --badge-color:#ff5677;
   --hero-color:#282c3f;
+  --grey-text:#C0A080;
```
Comment (mayuresh-tech) [inline css/main.css:30]: same suggestion: variables can be added

**Review by mayuresh-tech — COMMENTED** [review]
Else everything looks good!
Good to merge

File: css/main.css (hunk 2 above)
Comment (BALAJI141031) [reply css/main.css:30]: these are variables only and i'm using this in other files

**Pull request merged**